rs/registry/canister/src/common/test_helpers.rs

```diff
@@ -115,7 +116,7 @@ pub fn prepare_registry_with_nodes(
                     ip_addr: format!("128.0.{effective_id}.1"),
                     ..Default::default()
                 }),
-                node_operator_id: PrincipalId::new_user_test_id(999).into_vec(),
```

Comment:
This is a behavior change in the case where the user passes "None".  default is different than PrincipalId::new_user_test_id(999).